### PR TITLE
Luckydonald superduper progressbar branch thingie

### LINKED ISSUE #8105: Enhancement: Show "time left" during upload in Web UI.
Would be nice to have some information about how long time there is left during upload in the Web UI.

This is good for example when you´re on a slow internet connection and is uploading many files at the same time. Or for big files uploads.

Anyway, would be nice to have.

### PR DESCRIPTION
**Pull request by PVince81**
From https://github.com/owncloud/core/pull/10326 Fixes https://github.com/owncloud/core/issues/8105

Thanks a lot @luckydonald 

Would be good if people could give it a quick retest to make sure it still works.

I tested in Firefox and it seemed to work fine, although my computer/SSD is so fast I could only ever see the "2 seconds left" message.

:+1: from me

Second review @rullzer @blizzz or others ?

### CODE REVIEW

**Comment by PVince81** [comment]
License is MIT as stated here https://github.com/owncloud/core/pull/10326#issuecomment-218702773

**Comment by davitol** [comment]
I was not able to see the "..seconds left" message in the progress bar or as a tooltip on it. Tested with Chrome and Firefox for MacOSX

**Comment by PVince81** [comment]
@davitol you need a slower computer, maybe a slow proxy of some sorts.

**Comment by MorrisJobke** [comment]
Works here just fine 👍

File: apps/files/css/mobile.css [hunk 1]
```diff
@@ -67,7 +67,15 @@ table td.filename .nametext .innernametext {
...
 }
+/* hide desktop-only parts */
+#uploadprogressbar .desktop {
+	display: none !important;
 }
+#uploadprogressbar .mobile {
+	display: block !important;
+}
+
+}
```
Comment (nickvergessen) [inline apps/files/css/mobile.css]: accident?

Comment (PVince81) [inline apps/files/css/mobile.css]: possibly yes when resolving the conflict, let me adjust this

Comment (PVince81) [inline apps/files/css/mobile.css]: ah no, it's the end of the media query element

**Comment by MorrisJobke** [comment]
@PVince81 Could you hit the merge button? This is nearly only JS (and some template changes in a PHP file) and the JSUnit job was successful :)

Thanks

**Comment by PVince81** [comment]
Did you guys also test folder upload with chrome ? I realized I forgot to test that part.
Trying nowq

**Comment by PVince81** [comment]
Ok, folder upload still works.

**Comment by luckydonald** [comment]
Awesome! thank you guys.

**Pull request merged**